src/pytest_postgresql/janitor.py

```diff
@@ -23,7 +23,8 @@ def __init__(
             port: str,
             db_name: str,
             version: Union[str, float, Version],
-            password: str = None
+            password: str = None,
+            isolation_level: int = 0,
```

Comment:
can we use psycopg2 constants or were there some issues?

Reply:
Good call, I'll get that switched up shortly.